gr-blocks/examples/msg_passing/hier/test_msg_hier.grc

```diff
@@ -118,16 +136,18 @@ blocks:
     type: message
     vlen: '1'
   states:
+    bus_sink: false
+    bus_source: false
+    bus_structure: null
     coordinate: [120, 148.0]
     rotation: 0
     state: enabled
 
 connections:
-- [blocks_message_strobe_0, strobe, pad_sink_0, in0]
-- [blocks_message_strobe_0_1, strobe, pad_sink_0, in0]
```

Comment:
Hmm, did `blocks_message_strobe_0_1` really go to `pad_sink_0` as well? Any idea why this may make sense?

Reply:
Ok, I checked with the version in maint-3.7 and it should now be correct. There were indeed some "funny" connections.